public/src/components/forms/GroupForm.tsx

```diff
@@ -281,8 +278,13 @@ export class GroupForm extends React.Component<IGroupProps, IGroupState> {
     }
 
     private isCurrentStudentSelected(student: IUserRelation): boolean {
-        const index = this.state.selectedStudents.indexOf(student);
-        return index >= 0;
+        let ans = false;
```

Comment:
It would be better to use `found` or `foundSelected` instead of `ans`.

Reply:
Fixed.